### PR TITLE
Brush up Gourd Leshy Heritage's Perception Bonus against Thievery checks

### LINKED ISSUE #16969: Issue with Gourd Leshy - receiving bonus to perception/initiative?
image

There's something weird going on with Gourd Leshy. We can't figure out why the level 2 character is being granted a +4. As far as I can find, there's nothing on AoN called "Gourd Head" at all, let alone anything that would give a Gourd Leshy a +4 to initiative rolls.

### PR DESCRIPTION
**Pull request by ammalagonc**
So, I did a bit of a hack to avoid the bonus reflecting on initiative rolls. I'm not a fan of the toggle, per se, so if there's a better way to handle this, by all means, let me know.

Closes #16969

### CODE REVIEW

**Comment by simonward** [comment]
Does a selector of "perception-dc" work?

**Comment by ammalagonc** [comment]
Tried it - nada :|

**Comment by pedrogrullada** [comment]
`perception-dc` as a selector works for me, and then adding `action:steal` as a predicate to narrow it down further

**Comment by ammalagonc** [comment]
OK, now it works

I'm getting rid of this toggle!

**Comment by pedrogrullada** [comment]
> OK, now it works
> 
> I'm getting rid of this toggle!

How so? Not everything gets stolen from your head!

**Comment by simonward** [comment]
> I'm getting rid of this toggle!

Unfortunately, it might still be needed, as it specifies stealing from your head.

**Comment by ammalagonc** [comment]
Toggle lives...
For now

File: packs/heritages/leshy/gourd-leshy.json [hunk 1]
```diff
@@ -28,10 +28,11 @@
                 "key": "FlatModifier",
                 "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
                 "predicate": [
+                    "action:steal",
                     "gourd-head"
                 ],
-                "selector": "perception",
-                "type": "untyped",
+                "selector": "perception-dc",
+                "slug": "gourd-head-bonus",
```
Comment (simonward) [inline packs/heritages/leshy/gourd-leshy.json:35]:
```suggestion
```

**Comment by simonward** [comment]
Consider updating both Bottlespeakers.

File: packs/heritages/leshy/gourd-leshy.json [hunk 2]
```diff
@@ -22,16 +22,17 @@
                 "domain": "all",
                 "key": "RollOption",
                 "option": "gourd-head",
+                "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
                 "toggleable": true
             },
             {
                 "key": "FlatModifier",
                 "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
```
Comment (simonward) [inline packs/heritages/leshy/gourd-leshy.json:30]:
```suggestion
```

File: packs/paizo-pregens/bottlespeaker-level-4.json [hunk 3]
```diff
@@ -17,28 +17,29 @@
...
                     {
                         "domain": "all",
                         "key": "RollOption",
+                        "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
                         "option": "gourd-head",
                         "toggleable": true
                     },
                     {
                         "key": "FlatModifier",
                         "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
```
Comment (simonward) [inline packs/paizo-pregens/bottlespeaker-level-4.json:37]:
```suggestion
```

File: packs/paizo-pregens/bottlespeaker.json [hunk 4]
```diff
@@ -17,28 +17,29 @@
...
                     {
                         "domain": "all",
                         "key": "RollOption",
+                        "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
                         "option": "gourd-head",
                         "toggleable": true
                     },
                     {
                         "key": "FlatModifier",
                         "label": "PF2E.SpecificRule.ToggleProperty.GourdHead",
```
Comment (simonward) [inline packs/paizo-pregens/bottlespeaker.json:37]:
```suggestion
```

**Pull request merged**